python/nav/portadmin/handlers.py

```diff
@@ -0,0 +1,204 @@
+#
+# Copyright (C) 2011-2015, 2020 UNINETT
+#
+# This file is part of Network Administration Visualized (NAV).
+#
+# NAV is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License version 3 as published by
+# the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+# details.  You should have received a copy of the GNU General Public License
+# along with NAV. If not, see <http://www.gnu.org/licenses/>.
+#
+"""Interface definition for PortAdmin management handlers"""
+from django.utils.encoding import python_2_unicode_compatible
+
+
+@python_2_unicode_compatible
```

Comment:
If `master` no longer cares about Python 2.7, you no longer need `@python_2_unicode_compatible` anywhere.

Reply:
Indeed :-)